packages/react/src/useFileUpload.ts

```diff
@@ -5,22 +5,24 @@ import {
   FileItemRef,
   FileUploadMachine,
   FileUploadState,
+  StorageUploadParams,
   UploadFileHandlerResult,
   uploadFilePromise
 } from '@nhost/hasura-storage-js'
 import { useInterpret, useSelector } from '@xstate/react'
 
 import { useNhostClient } from './useNhostClient'
+
```

Comment:
I can't leave comments on unchanged lines, but could you please fix the formatting of the `useFileUpload` example in this file? It gets formatted properly for auto-generated docs, but not for IDEs.

Reply:
I don't get the problem. Could you share a snapshot?

Reply:
<img width="553" alt="Screenshot 2022-06-29 at 14 00 21" src="https://user-images.githubusercontent.com/310881/176431323-8b9e49c3-c056-4f32-84c3-567176c0d774.png">

it should be displayed like this:

```ts
const {
  add,
  upload,
  cancel,
  // rest...
} = useFileUpload();
```